File: app/src/main/java/com/rinconadadelsur/sistemas/bioseguridad/Entidades/eRegistroGarita.java
package com.rinconadadelsur.sistemas.bioseguridad.Entidades;

public class eRegistroGarita {

    private String _codigo;
    private String _anomalia;
    private String _descripcion;

    public eRegistroGarita(){

    }

    public eRegistroGarita(String _codigo, String _anomalia, String _descripcion ) {
        this._codigo = _codigo;
        this._anomalia = _anomalia;
        this._descripcion = _descripcion;
    }

    public String get_codigo() {
        return _codigo;
    }

    public void set_codigo(String _codigo) {
        this._codigo = _codigo;
    }

    public String get_anomalia() {
        return _anomalia;
    }

    public void set_anomalia(String _anomalia) {
        this._anomalia = _anomalia;
    }

    public String get_descripcion() {
        return _descripcion;
    }

    public void set_descripcion(String _descripcion) {
        this._descripcion = _descripcion;
    }

    @Override
    public String toString() {
        return _descripcion;
    }
}

File: app/src/main/java/com/rinconadadelsur/sistemas/bioseguridad/Entidades/eRegistroGarita.kt
package com.rinconadadelsur.sistemas.bioseguridad.Entidades

class eRegistroGarita {
    private var _codigo: String? = null
    private var _anomalia: String? = null
    private var _descripcion: String? = null

    constructor()

    constructor(_codigo: String?, _anomalia: String?, _descripcion: String) {
        this._codigo = _codigo
        this._anomalia = _anomalia
        this._descripcion = _descripcion
    }

    fun get_codigo(): String? {
        return _codigo
    }

    fun set_codigo(_codigo: String?) {
        this._codigo = _codigo
    }

    fun get_anomalia(): String? {
        return _anomalia
    }

    fun set_anomalia(_anomalia: String?) {
        this._anomalia = _anomalia
    }

    fun get_descripcion(): String {
        return _descripcion!!
    }

    fun set_descripcion(_descripcion: String) {
        this._descripcion = _descripcion
    }

    override fun toString(): String {
        return _descripcion!!
    }
}
